build-info-extractor/src/main/java/org/jfrog/build/extractor/clientConfiguration/client/ArtifactoryBuildInfoClient.java

```diff
@@ -190,6 +188,11 @@ public void sendBuildInfo(String buildInfoJson) throws IOException {
     }
 
     public Build getBuildInfo(String buildName, String buildNumber) throws IOException {
+        buildNumber = getLatestBuildNumberFromArtifactory(buildName, buildNumber);
```

Comment:
I suggest adding the following comment above this line:
// Only If the value of the buildNumber sent is "LATEST" or "LAST_RELEASE", replace the value with a specific build number.